
# HG changeset patch
# User Makoto Kato <m_kato@ga2.so-net.ne.jp>
# Date 1510284876 -32400
# Node ID 7dabae5e261a121818cf7ce8e06fe91bd68a4913
# Parent  f962fb3449f44bdbe51299b8fa9b3b5a4970321b
Bug 1415014 - Part 1. Add default64.png and default128.png for GTK HiDPI icons. r=dao

Now, default48.png is highest resolition icon for Firfox/GTK.  So we should add
default64.png (from icon64.png) and default128.png (from mozicon128.png) for
HiDPI.  Also mozicon128.png is for Linux distribution, we should move to
correct place for our GTK icons.

MozReview-Commit-ID: BeLXzUcOgQu

diff --git a/browser/branding/aurora/content/jar.mn b/browser/branding/aurora/content/jar.mn
--- a/browser/branding/aurora/content/jar.mn
+++ b/browser/branding/aurora/content/jar.mn
@@ -3,15 +3,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 browser.jar:
 % content branding %content/branding/ contentaccessible=yes
   content/branding/about.png
   content/branding/about-logo.png
   content/branding/about-logo@2x.png
   content/branding/about-wordmark.svg
-  content/branding/icon48.png
-  content/branding/icon64.png
   content/branding/icon16.png                    (../default16.png)
   content/branding/icon32.png                    (../default32.png)
-  content/branding/icon128.png                   (../mozicon128.png)
+  content/branding/icon48.png
+  content/branding/icon64.png                    (../default64.png)
+  content/branding/icon128.png                   (../default128.png)
   content/branding/identity-icons-brand.svg
   content/branding/aboutDialog.css
diff --git a/browser/branding/aurora/mozicon128.png b/browser/branding/aurora/default128.png
rename from browser/branding/aurora/mozicon128.png
rename to browser/branding/aurora/default128.png
diff --git a/browser/branding/aurora/content/icon64.png b/browser/branding/aurora/default64.png
rename from browser/branding/aurora/content/icon64.png
rename to browser/branding/aurora/default64.png
diff --git a/browser/branding/branding-common.mozbuild b/browser/branding/branding-common.mozbuild
--- a/browser/branding/branding-common.mozbuild
+++ b/browser/branding/branding-common.mozbuild
@@ -37,19 +37,21 @@ def FirefoxBranding():
             'background.png',
             'disk.icns',
             'document.icns',
             'dsstore',
             'firefox.icns',
         ]
     elif 'gtk' in CONFIG['MOZ_WIDGET_TOOLKIT']:
         BRANDING_FILES += [
+            'default128.png',
             'default16.png',
             'default32.png',
             'default48.png',
-            'mozicon128.png',
+            'default64.png',
         ]
-        FINAL_TARGET_FILES.icons += ['mozicon128.png']
         FINAL_TARGET_FILES.chrome.icons.default += [
+            'default128.png',
             'default16.png',
             'default32.png',
             'default48.png',
+            'default64.png',
         ]
diff --git a/browser/branding/nightly/content/jar.mn b/browser/branding/nightly/content/jar.mn
--- a/browser/branding/nightly/content/jar.mn
+++ b/browser/branding/nightly/content/jar.mn
@@ -3,15 +3,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 browser.jar:
 % content branding %content/branding/ contentaccessible=yes
   content/branding/about.png
   content/branding/about-logo.png
   content/branding/about-logo@2x.png
   content/branding/about-wordmark.svg
-  content/branding/icon48.png
-  content/branding/icon64.png
   content/branding/icon16.png                    (../default16.png)
   content/branding/icon32.png                    (../default32.png)
-  content/branding/icon128.png                   (../mozicon128.png)
+  content/branding/icon48.png
+  content/branding/icon64.png                    (../default64.png)
+  content/branding/icon128.png                   (../default128.png)
   content/branding/identity-icons-brand.svg
   content/branding/aboutDialog.css
diff --git a/browser/branding/nightly/mozicon128.png b/browser/branding/nightly/default128.png
rename from browser/branding/nightly/mozicon128.png
rename to browser/branding/nightly/default128.png
diff --git a/browser/branding/nightly/content/icon64.png b/browser/branding/nightly/default64.png
rename from browser/branding/nightly/content/icon64.png
rename to browser/branding/nightly/default64.png
diff --git a/browser/branding/official/content/jar.mn b/browser/branding/official/content/jar.mn
--- a/browser/branding/official/content/jar.mn
+++ b/browser/branding/official/content/jar.mn
@@ -3,15 +3,15 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 browser.jar:
 % content branding %content/branding/ contentaccessible=yes
   content/branding/about.png
   content/branding/about-logo.png
   content/branding/about-logo@2x.png
   content/branding/about-wordmark.svg
-  content/branding/icon48.png
-  content/branding/icon64.png
   content/branding/icon16.png                    (../default16.png)
   content/branding/icon32.png                    (../default32.png)
+  content/branding/icon48.png
+  content/branding/icon64.png                    (../default64.png)
   content/branding/icon128.png                   (../mozicon128.png)
   content/branding/identity-icons-brand.svg
   content/branding/aboutDialog.css
diff --git a/browser/branding/official/mozicon128.png b/browser/branding/official/default128.png
rename from browser/branding/official/mozicon128.png
rename to browser/branding/official/default128.png
diff --git a/browser/branding/official/content/icon64.png b/browser/branding/official/default64.png
rename from browser/branding/official/content/icon64.png
rename to browser/branding/official/default64.png
diff --git a/browser/branding/unofficial/content/jar.mn b/browser/branding/unofficial/content/jar.mn
--- a/browser/branding/unofficial/content/jar.mn
+++ b/browser/branding/unofficial/content/jar.mn
@@ -4,15 +4,15 @@
 
 browser.jar:
 % content branding %content/branding/ contentaccessible=yes
   content/branding/about.png
   content/branding/about-background.png
   content/branding/about-logo.png
   content/branding/about-logo@2x.png
   content/branding/about-wordmark.svg
-  content/branding/icon48.png
-  content/branding/icon64.png
   content/branding/icon16.png                    (../default16.png)
   content/branding/icon32.png                    (../default32.png)
-  content/branding/icon128.png                   (../mozicon128.png)
+  content/branding/icon48.png
+  content/branding/icon64.png                    (../default64.png)
+  content/branding/icon128.png                   (../default128.png)
   content/branding/identity-icons-brand.svg
   content/branding/aboutDialog.css
diff --git a/browser/branding/unofficial/mozicon128.png b/browser/branding/unofficial/default128.png
rename from browser/branding/unofficial/mozicon128.png
rename to browser/branding/unofficial/default128.png
diff --git a/browser/branding/unofficial/content/icon64.png b/browser/branding/unofficial/default64.png
rename from browser/branding/unofficial/content/icon64.png
rename to browser/branding/unofficial/default64.png
diff --git a/browser/installer/allowed-dupes.mn b/browser/installer/allowed-dupes.mn
--- a/browser/installer/allowed-dupes.mn
+++ b/browser/installer/allowed-dupes.mn
@@ -71,16 +71,18 @@ modules/devtools/Console.jsm
 modules/devtools/Loader.jsm
 modules/devtools/shared/Console.jsm
 modules/devtools/shared/Loader.jsm
 browser/modules/devtools/client/framework/gDevTools.jsm
 browser/modules/devtools/gDevTools.jsm
 browser/chrome/icons/default/default16.png
 browser/chrome/icons/default/default32.png
 browser/chrome/icons/default/default48.png
+browser/chrome/icons/default/default64.png
+browser/chrome/icons/default/default128.png
 browser/chrome/pdfjs/content/web/images/findbarButton-next-rtl.png
 browser/chrome/pdfjs/content/web/images/findbarButton-next-rtl@2x.png
 browser/chrome/pdfjs/content/web/images/findbarButton-next.png
 browser/chrome/pdfjs/content/web/images/findbarButton-next@2x.png
 browser/chrome/pdfjs/content/web/images/findbarButton-previous-rtl.png
 browser/chrome/pdfjs/content/web/images/findbarButton-previous-rtl@2x.png
 browser/chrome/pdfjs/content/web/images/findbarButton-previous.png
 browser/chrome/pdfjs/content/web/images/findbarButton-previous@2x.png
@@ -100,17 +102,16 @@ browser/features/flyweb@mozilla.org/chro
 browser/features/flyweb@mozilla.org/chrome/skin/osx/icon-32-anchored.png
 browser/features/flyweb@mozilla.org/chrome/skin/osx/icon-64-anchored.png
 browser/features/flyweb@mozilla.org/chrome/skin/windows/flyweb.css
 browser/features/flyweb@mozilla.org/chrome/skin/windows/icon-16.png
 browser/features/flyweb@mozilla.org/chrome/skin/windows/icon-32-anchored.png
 browser/features/flyweb@mozilla.org/chrome/skin/windows/icon-32.png
 browser/features/flyweb@mozilla.org/chrome/skin/windows/icon-64-anchored.png
 browser/features/flyweb@mozilla.org/chrome/skin/windows/icon-64.png
-browser/icons/mozicon128.png
 chrome.manifest
 chrome/toolkit/skin/classic/global/autocomplete.css
 chrome/toolkit/skin/classic/global/button.css
 chrome/toolkit/skin/classic/global/checkbox.css
 chrome/toolkit/skin/classic/global/dialog.css
 chrome/toolkit/skin/classic/global/dropmarker.css
 chrome/toolkit/skin/classic/global/global.css
 chrome/toolkit/skin/classic/global/groupbox.css
diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -610,16 +610,18 @@
 @RESPATH@/chrome/toolkit@JAREXT@
 @RESPATH@/chrome/toolkit.manifest
 @RESPATH@/chrome/recording.manifest
 @RESPATH@/chrome/recording/*
 #ifdef MOZ_GTK
 @RESPATH@/browser/chrome/icons/default/default16.png
 @RESPATH@/browser/chrome/icons/default/default32.png
 @RESPATH@/browser/chrome/icons/default/default48.png
+@RESPATH@/browser/chrome/icons/default/default64.png
+@RESPATH@/browser/chrome/icons/default/default128.png
 #endif
 @RESPATH@/browser/features/*
 
 ; [Webide Files]
 @RESPATH@/browser/chrome/webide@JAREXT@
 @RESPATH@/browser/chrome/webide.manifest
 @RESPATH@/browser/@PREF_DIR@/webide-prefs.js
 
@@ -632,18 +634,16 @@
 @RESPATH@/browser/chrome/devtools@JAREXT@
 @RESPATH@/browser/chrome/devtools.manifest
 @RESPATH@/browser/@PREF_DIR@/devtools.js
 @RESPATH@/browser/@PREF_DIR@/debugger.js
 
 ; shell icons
 #ifdef XP_UNIX
 #ifndef XP_MACOSX
-; shell icons
-@RESPATH@/browser/icons/*.png
 #ifdef MOZ_UPDATER
 ; updater icon
 @RESPATH@/icons/updater.png
 #endif
 #endif
 #endif
 
 ; [Default Preferences]

